packages/react/src/components/CodeSnippet/CodeSnippet.mdx

```diff
@@ -0,0 +1,54 @@
+import { Story, Props, Source, Preview } from '@storybook/addon-docs/blocks'
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs'
+import * as stories from './CodeSnippet-story.js'
+import CodeSnippet from '../CodeSnippet'
+
+<!-- How do I get <Props> to work?
+ -->
+# CodeSnippet
+
+## Overview
+Code snippets are strings or small blocks of reusable code that can be copied and inserted in a code file.
+
+<Preview>
+	<CodeSnippet>
```

Comment:
For snippets I think we've been trying to define them over in the story file (`CodeSnippet-story.js` )and then refer to them with the `<Story id="..." />` convention 👀